README.md

```diff
@@ -1,22 +1,21 @@
-# quickstart-atlassian-jira
-## Jira Software and Jira Service Desk Data Center on the AWS Cloud
+# quickstart-atlassian-crowd
+## Crowd Software Data Center on the AWS Cloud
 
-Use this Quick Start to deploy one of the following Jira products from Atlassian on the AWS Cloud:
+Use this Quick Start to deploy Crowd Data Center on the AWS Cloud.
 
-* Jira Software Data Center
-* Jira Service Desk Data Center
+Crowd enables you to manage users from multiple directories - Active Directory, LDAP, OpenLDAP or Microsoft Azure AD - and control application authentication permissions in one single location. Crowd Data Center is a self-managed solution that gives you high availability, performance at scale, and disaster recovery for uninterrupted access to Crowd for all your teams.
 
-This Quick Start uses the [Atlassian Standard Infrastructure (ASI)](https://fwd.aws/xYyYy) as a foundation. You can choose to build a new ASI for your deployment or deploy Jira into your existing ASI. You can also deploy Bitbucket Data Center and Confluence Data Center within the same ASI.
+This Quick Start uses the [Atlassian Standard Infrastructure (ASI)](https://fwd.aws/xYyYy) as a foundation. You can choose to build a new ASI for your deployment or deploy Confluence into your existing ASI. You can also deploy Jira, Confluence and Bitbucket Data Center within the same ASI.
```

Comment:
Should this say "or deploy **Crowd** into your existing ASI"?

Reply:
It should indeed, cheers.